src/t_zset.c

```diff
@@ -4035,11 +4062,11 @@ void genericZpopCommand(client *c, robj **keyv, int keyc, int where, int emitkey
             serverAssertWithInfo(c,zobj,zln != NULL);
             ele = sdsdup(zln->ele);
```

Comment:
Another minor optimization: for the skiplist, we don't need to copy the ele. Or perhaps we should change it so that listpack and skiplist handle their own loops respectively.